Project3/index5.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">  
+    <title>10:00 A.M.</title>
+    <div class="card">
+    <h1>10:00 A.M.</h1>
+    <h2>Stop Watching Netflix/YouTube</h2></head>
+<body style="background-color:#000000"; >
+    <p>Hi my name's Timmy and I'm addicted to watching Netflix & YouTube.<br />
+        I want to be remembered for more in my life.<br />
```

Comment:
I think about this a lot after losing hours into a game or an internet spiral, too 🤦‍♀